source/adios2/toolkit/sst/dp/ucx_dp.c

```diff
@@ -372,9 +461,9 @@ static void UcxProvideWriterDataToReader(CP_Services Svcs, DP_RS_Stream RS_Strea
         if (status != UCS_OK)
         {
             Svcs->verbose(RS_Stream->CP_Stream, DPCriticalVerbose,
-                          "UCX Error during ucp_ep_create() with: %s.\n",
+                          "UCX Error during ucp_ep_create() with: %s. Let's ignore for now, this "
+                          "point-to-point connection might not be needed.\n",
```

Comment:
This might be a scaling bug in the UCX backend: It creates endpoints to all existing writers during initialization. If using UCX for shared-memory-only communication on multi-node setups, this will lead to failures here, but we can ignore them up to the point where actual data is loaded from that endpoint.